l1-contracts/contracts/common/Config.sol

```diff
@@ -103,7 +103,7 @@ uint256 constant MEMORY_OVERHEAD_GAS = 10;
 uint256 constant PRIORITY_TX_MAX_GAS_LIMIT = 72_000_000;
 
 /// @dev the address used to identify eth as the base token for chains.
-address constant ETH_TOKEN_ADDRESS = address(1);
+address constant BASE_TOKEN_VIRTUAL_ADDRESS = address(1);
```

Comment:
I am a bit against calling it any differently than ETH_TOKEN_ADDRESS, because:

a) it is only ETH today
b) I am not sure if we will ever use `address(1)` for anything except for ETH. I think the only reason this address even exists is because previously we used token addresses instead for asset ids, and so for ETH it was address(1). If we started with asset ids from the start we would supply something like `bytes32(1)` or maybe calculate it in some other way
c) It is currently a hardcoded constant in Config, I am not sure it will work fine if we use the same constant for each chain's base token

Reply:
Ok, we should keep it like it is for now. 

With interop... I'm not sure, we might need it, but assetId might be enough as you said.